From c49f1cea96246bdd47b7d57bae399fd7a21f7089 Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Sat, 5 Dec 2020 20:42:39 +0530
Subject: [PATCH 18/40] Revert "ANDROID: x86/extable: Do not mark exception
 callback as CFI"

This reverts commit fea23ded4601fe9777134fba2d7323f86dd48062.
---
 arch/x86/mm/extable.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/mm/extable.c b/arch/x86/mm/extable.c
index 3cacb49a54e8..4d75bc656f97 100644
--- a/arch/x86/mm/extable.c
+++ b/arch/x86/mm/extable.c
@@ -199,7 +199,6 @@ __visible bool ex_has_fault_handler(unsigned long ip)
 	return handler == ex_handler_fault;
 }
 
-__nocfi
 int fixup_exception(struct pt_regs *regs, int trapnr, unsigned long error_code,
 		    unsigned long fault_addr)
 {
-- 
2.17.1

